src/platform/tests/TestKeyValueStoreMgr.cpp

```diff
@@ -260,6 +260,7 @@ static void TestKeyValueStoreMgr_NonExistentDelete(nlTestSuite * inSuite, void *
     NL_TEST_ASSERT(inSuite, err == CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND);
 }
 
+#if !defined(__ZEPHYR__) && !defined(__MBED__)
```

Comment:
Why?  Does the test fail there?  Are there issues tracking the failure?  They should be mentioned here, if so.

Reply:
This fixes the unused-function build issue that occurs with ARC GCC 10.3.1. The same condition is in line 322.